includes/class-civicrm-caldera-forms-helper.php

```diff
@@ -484,13 +484,43 @@ public function map_fields_to_prerender( $config, &$form, $ignore_fields, $entit
 				if ( $field['type'] == 'radio' ) {
 					$options = Caldera_Forms_Field_Util::find_option_values( $field );
 					$form['fields'][$field['ID']]['config']['default'] = array_search( $value, $options );
+				} elseif ( $field['type'] == 'date_picker' ) {
+					// If the date is still in the format returned by the CiviCRM API, convert it to the date picker's format.
+					$filtered_value = &$form['fields'][$field['ID']]['config']['default'];
+					if ( preg_match('/^\d{4}-\d{2}-\d{2}(?: \d{2}:\d{2}:\d{2})?$/', $filtered_value ) ) {
```

Comment:
I don't think `preg_match` is necessary here, I think it's safe to format the date regardless, as long as we have a date, so to complete my comment above this could be simplified to:
``` php
if ( $field['type'] == 'date_picker' && ! empty( $value ) ) {
	$format = $this->translate_date_picker_format( $field['config']['format'] );
	$value = date_create( $value )->format( $format );
}
```

Reply:
Will do.